Error running "make docs": « exception: No module named 'sphinx_multiversion' »

Trying to run "make docs" on a fresh clone of the wire-docs repo ( docker was installed with «sudo apt install docker.io», otherwise this is a fresh Debian 10 install ), I get the following error:

```
arthur@debian-wire:~/jct-9$ git clone https://github.com/wireapp/wire-docs.git
Cloning into 'wire-docs'...
remote: Enumerating objects: 1685, done.
remote: Counting objects: 100% (350/350), done.
remote: Compressing objects: 100% (184/184), done.
remote: Total 1685 (delta 183), reused 282 (delta 152), pack-reused 1335
Receiving objects: 100% (1685/1685), 8.74 MiB | 3.66 MiB/s, done.
Resolving deltas: 100% (1017/1017), done.
arthur@debian-wire:~/jct-9$ cd wire-docs/
arthur@debian-wire:~/jct-9/wire-docs$ ls
ci  Dockerfile  Dockerfile-poetry  Makefile  nix  poetry.lock  pyproject.toml  README.md  shell.nix  src
arthur@debian-wire:~/jct-9/wire-docs$ make docs
docker run --rm -v $(pwd):/mnt quay.io/wire/alpine-sphinx:latest make clean html
rm -rf "/mnt/build"
sphinx-build -M html "/mnt/src" "/mnt/build" -q

Extension error:
Could not import extension sphinx_multiversion (exception: No module named 'sphinx_multiversion')
make: *** [Makefile:108: html] Error 2
make: *** [Makefile:31: docs] Error 2
arthur@debian-wire:~/jct-9/wire-docs$ 
```

Any ideas why I am getting this error, and what I can do to solve the issue? 
Should I be trying to run a different branch?

--- a/Makefile
+++ b/Makefile
@@ -6,7 +6,7 @@ MKFILE_DIR = $(abspath $(dir $(abspath $(lastword $(MAKEFILE_LIST)))))
 
 DOCKER_USER   ?= quay.io/wire
 DOCKER_IMAGE  = alpine-sphinx
-DOCKER_TAG    ?= latest
+DOCKER_TAG    ?= 0.0.32
 
 # You can set these variables (with a ?=) from the command line, and also
 # from the environment.

